src/lazy/encoder/binary/v1_0/mod.rs

```diff
@@ -14,8 +13,4 @@ impl LazyEncoder for BinaryEncoding_1_0 {
         SymbolCreationPolicy::RequireSymbolId;
 
     type Writer<W: Write> = LazyRawBinaryWriter_1_0<W>;
-
-    fn default_write_config() -> WriteConfig<Self> {
```

Comment:
🗺️ The `Encoding` trait (which was previously mostly used as a marker trait) now extends `LazyEncoder` and `LazyDecoder`, meaning that we can start using `Encoding` everywhere instead of the pair. As part of this reorganization, the default write config method now lives on `Encoding`; `Encoding` can refer to the `LazyEncoder`, but the `LazyEncoder` cannot refer to `Encoding`.